pkg/blobinfocache/internal/test/test.go

```diff
@@ -103,6 +116,14 @@ func assertCandidatesMatch(t *testing.T, scopeName string, expected []candidate,
 	assert.Equal(t, e, actual)
 }
 
+func assertCandidatesMatch2(t *testing.T, scopeName string, expected []candidate, actual []types.BICReplacementCandidate2) {
```

Comment:
If the set of new tests is the same as the old tests (is it?), maybe instead move the `cache.CandidateLocations{,2}` calls inside the `assert…` helper, have a single helper call test both methods, and we can avoid the duplication.

(This can clearly wait until after the interface is settled.)